samples/nrf9160/spm/README.rst

```diff
@@ -30,6 +30,17 @@ Requirements for the application firmware
 
 * The application firmware must be built as a non-secure firmware for the nrf9160_pca10090ns board.
 
+Automatic building of SPM
+=========================
+
+The sample is automatically built by non-secure applications when the nrf9160_pca10090ns board is used.
```

Comment:
Should we, perhaps, explicitly stress that the "sample is built by the non-secure applications, when...." but the sample is not part of the non-secure application....

Reply:
Tried to reword - please check again.